### ISSUE TITLE
Nightly performance_test job failing

### ISSUE THREAD

**Issue by ivanpauno** [ros2/performance_test#11]
See for example:

- http://build.ros2.org/view/Fci/job/Fci__nightly-performance_ubuntu_focal_amd64/47/consoleFull#console-section-26

<details>
<summary>Error output</summary>

```
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp: In member function ‘eprosima::fastrtps::Participant* performance_test::ResourceManager::fastrtps_participant() const’:
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:81:23: error: ‘class eprosima::fastrtps::rtps::BuiltinAttributes’ has no member named ‘use_SIMPLE_RTPSParticipantDiscoveryProtocol’
07:20:43    81 |   PParam.rtps.builtin.use_SIMPLE_RTPSParticipantDiscoveryProtocol = true;
07:20:43       |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:82:23: error: ‘class eprosima::fastrtps::rtps::BuiltinAttributes’ has no member named ‘use_SIMPLE_EndpointDiscoveryProtocol’
07:20:43    82 |   PParam.rtps.builtin.use_SIMPLE_EndpointDiscoveryProtocol = true;
07:20:43       |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:83:23: error: ‘class eprosima::fastrtps::rtps::BuiltinAttributes’ has no member named ‘m_simpleEDP’
07:20:43    83 |   PParam.rtps.builtin.m_simpleEDP.use_PublicationReaderANDSubscriptionWriter = true;
07:20:43       |                       ^~~~~~~~~~~
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:84:23: error: ‘class eprosima::fastrtps::rtps::BuiltinAttributes’ has no member named ‘m_simpleEDP’
07:20:43    84 |   PParam.rtps.builtin.m_simpleEDP.use_PublicationWriterANDSubscriptionReader = true;
07:20:43       |                       ^~~~~~~~~~~
07:20:43 /tmp/ws2/src/performance_test/performance_test/src/communication_abstractions/resource_manager.cpp:85:23: error: ‘class eprosima::fastrtps::rtps::BuiltinAttributes’ has no member named ‘leaseDuration’
07:20:43    85 |   PParam.rtps.builtin.leaseDuration = eprosima::fastrtps::c_TimeInfinite;
```
</details>

@MiguelCompany the build error seems to come from an API change in `Fast-RTPS`.
Was it intentional and this repo should be updated or should it be fixed in `fastrtps`?

**Comment by ivanpauno** [comment]
I think https://github.com/ros2/performance_test/pull/12 fixed this:

https://github.com/ros2/performance_test/blob/437e037015ddccfb875747839f5481140c2fa8aa/performance_test/src/communication_abstractions/resource_manager.cpp#L75-L87

I will close this issue if today's CI passes.

**Comment by ivanpauno** [comment]
Fixed in #12, #14.
